api/v1alpha1/driver_types.go

```diff
@@ -28,9 +28,17 @@ type LogSpec struct {
 	// Log level for driver pods,
 	// Supported values from 0 to 5. 0 for general useful logs (the default), 5 for trace level verbosity.
 	// Default to 0
-	LogLevel   int               `json:"logLevel,omitempty"`
-	MaxFiles   int               `json:"maxFiles,omitempty"`
+	LogLevel int `json:"logLevel,omitempty"`
+	// MaxFiles is the number of logrtoate files
+	MaxFiles int `json:"maxFiles,omitempty"`
+	// MaxLogSize is the maximum size of the log per csi pods.
 	MaxLogSize resource.Quantity `json:"maxLogSize,omitempty"`
+	// Periodicity is the periodicity of the log rotation.
+	// +kubebuilder:validation:Pattern=`^$|^(hourly|daily|weekly|monthly|1h|24h|1d)$`
```

Comment:
i dont think this is the schedule restriction from the logrotation, we need to have the same flexibility what we have with logrotation of support only few like `hourly|daily|weekly|monthly` not with hout like `1h,24h,1d` etc.